licenses/ccbyncnd-4.0.txt

```diff
@@ -0,0 +1,95 @@
+Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International
```

Comment:
[A plaintext version of the license I found](https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode.txt) has an extra section at the top, but I dunno if others have this and we don't routinely include them?

Reply:
Thanks for pointing this out @erinspace, looks like I was including too much.  [The other CC licenses](https://github.com/CenterForOpenScience/list-of-licenses/blob/master/licenses/ccby-4.0.txt) have this top section, but it doesn't look like we include them.  I guess it's not a part of the license, but more clarifying what Creative Commons is.